packages/editor/src/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -137,6 +153,14 @@ export function FillInTheBlanksRenderer(props: FillInTheBlanksRendererProps) {
           </BlankDraggableArea>
         ) : null}
 
+        {mode === 'drag-and-drop' && isEditing ? (
+          <BlankDraggableDummyArea
+            additionalDraggableAnswers={
+              additionalDraggableAnswers as FillInTheBlanksExerciseProps['state']['additionalDraggableAnswers']
```

Comment:
Because of the difference between editor and static plugin props.

I'd stick to the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), but if you insist, I can add the local type.

Reply:
Btw, if you have any ideas on how to avoid the type assertion, I'm all ears! 😁